scripts/set_xva_bridge.sh

```diff
@@ -0,0 +1,140 @@
+    echo "- Usage: $0 [XVA's_filename] compression[zstd|gzip] bridge_value[xenbr0|xapi[:9]|...]"
+    echo "- All options are mandatory."
+    echo "----------------------------------------------------------------------------------------"
+}
+
+# extract and read ova.xml
+info_ovaxml()
+{
+    TMPFOLD=$(mktemp -d /tmp/xvaXXXX)
+    tar -xvf ${XVA_NAME} -C ${TMPFOLD} ova.xml
+    chmod +r ${TMPFOLD}/ova.xml
+    XML_VALUE=$(grep -oE "<member><name>bridge</name><value>[^<]*</value></member>" ${TMPFOLD}/ova.xml)
+    LENGTH=${#XML_VALUE}
+    PREFIX_LENGTH=$((${LENGTH}-17))
+    NETWORK_VALUE=$(cut -c 1-${PREFIX_LENGTH} <<< ${XML_VALUE})
+    BRIDGE_READVALUE=$(cut -c 35-${#NETWORK_VALUE} <<< ${NETWORK_VALUE})
+
+    if [ -d ${TMPFOLD} ]; then
+        rm -Rf ${TMPFOLD}
+    fi
+}
+
+# check parameters and prompt the usage if needed
+if [ -z "${1+set}" ]
+then
+    echo "Error: XVA name missing."
+    usage
+    exit 1
+else
+    if [ -e "${1}" ]; then
+        XVA_NAME=$(realpath $1)
+
+        if [ ${XVA_NAME##*.} !=  xva ]
+        then
+            echo "Error: ${XVA_NAME} doesn't seem to be a xva file"
+            usage
+            exit 1
+        fi
+    else
+        echo "Error: $1 doesn't seem to exist as a file. Is the path or the name are corrects?"
+        usage
+        exit 1
+    fi
+fi
+
+if ([ "$2" = "gzip" ] || [ "$2" = "zstd" ])
+then
+    COMPRESS_METHOD=$2
+else
+    echo "Error: Wrong option for the compression."
+    usage
+    exit 1
+fi
+
+if [ -z "${3+set}" ]
+then
+    echo "Error: No entrie for the bridge's value."
+    usage
+    exit 1
+else
+    BRIDGE_VALUE=$3
+fi
+
+# we want to know the value of the network bridge.
+# then we can decide if we need to change it or not.
+info_ovaxml
+
+if [ -z "${BRIDGE_READVALUE}" ]
+then
+    echo "No bridge's value detected in the xml file!"
+    exit 1
+else
+    if [ ${BRIDGE_READVALUE} == ${BRIDGE_VALUE} ]
+    then
+        echo "It's already the ${BRIDGE_VALUE}. Nothing to do."
+        exit 1
+    fi
+fi
+
+# we detect the compression method of the xva to uncompress it right
+OLD_COMPRESSION=$(file ${XVA_NAME} | cut -f 2 -d :  | cut -f 2 -d " ")
+if [ ${OLD_COMPRESSION} = "Zstandard" ]; then
+    OPTIONUC="xf"
+elif [ ${OLD_COMPRESSION} = "gzip" ]; then
+    OPTIONUC="xzf"
+elif [ ${OLD_COMPRESSION} = "tar" ]; then
+    OPTIONUC="xf"
+else
+    echo "Error: The compression option of ${XVA_NAME} is unknow for this script: ${OLD_COMPRESSION}"
+    exit 1
+fi
+
+PATHFOLDER=$(dirname ${XVA_NAME})
+TMPFOLDER=$(mktemp -d ${PATHFOLDER}/xvaXXXX)
+
+if [ -f "${XVA_NAME}" ]; then
+    tar -${OPTIONUC} $1 -C "${TMPFOLDER}"
+else
+    echo "Error: ${XVA_NAME} not found."
+    exit 1
+fi
+
+if [ -e "${TMPFOLDER}/ova.xml" ]; then
+    chmod +rw ${TMPFOLDER}/ova.xml
+    sed -i "s/<member><name>bridge<\/name><value>[^<]*<\/value><\/member>/<member><name>bridge<\/name><value>${BRIDGE_VALUE}<\/value><\/member>/g" ${TMPFOLDER}/ova.xml
+    chmod -rw ${TMPFOLDER}/ova.xml
```

Comment:
```suggestion
    chmod +rw "${TMPFOLDER}/ova.xml"
    sed -i "s/<member><name>bridge<\/name><value>[^<]*<\/value><\/member>/<member><name>bridge<\/name><value>${BRIDGE_VALUE}<\/value><\/member>/g" "${TMPFOLDER}/ova.xml"
    chmod -rw "${TMPFOLDER}/ova.xml"
```

Maybe `"${TMPFOLDER}/ova.xml"` can be put in a var?